src/components/Button.tsx

```diff
@@ -12,7 +12,7 @@ interface Props {
   /**
    * button contents
    */
-  label: ReactNode;
+  children: ReactNode;
```

Comment:
using children for a component like this makes much more sense. No idea why I didn't do it like this in the first place